lib/broccoli/ember-addon.js

```diff
@@ -54,6 +54,12 @@ function EmberAddon() {
   }));
 }
 
+for (var key in EmberApp) {
```

Comment:
:+1: wasn't able to dig up whether it was supported in older versions of node but I tested it all the way back to 0.10.0 and seems to work.